src/emulator/eventarcEmulator.ts

```diff
@@ -76,7 +76,7 @@ export class EventarcEmulator implements EmulatorInstance {
       return { projectId, triggerName, eventTrigger, key };
     };
 
-    const removeTriggerRoute = `/emulator/v1/remove/projects/:project_id/triggers/:trigger_name`;
+    const removeTriggerRoute = `/emulator/v1/remove/projects/:project_id/triggers/:trigger_name(*)`;
```

Comment:
Quick sanity check since this seems like an unrelated bug fix - could you briefly explain why we made this change?

Reply:
Sure: During testing I noticed that removing eventarc triggers didn't work and always gave an error message. I eventually figured out that it was because the POST request was not getting to the removeTrigger handler, but rather hitting the fallthrough "*" handler instead. I investigated a bit more and realized it was because the trigger name has slashes (/) in it. So I looked at the addTriggerRoute (because that one was working) and saw the (*) on the end. Tried that same pattern for the removeTriggerRoute, and then it started working and removing the eventarc triggers when it was supposed to.